build/roundProgress.js

```diff
@@ -1,4 +1,4 @@
-/* angular-svg-round-progressbar@0.4.6 2016-07-06 */
+/* angular-svg-round-progressbar@0.4.6 2016-08-16 */
```

Comment:
This is an auto-generated file which shouldn't be touched manually. Check the previous comment.